lib/ui/plugins.dart

```diff
@@ -0,0 +1,74 @@
+// Copyright 2018 The Chromium Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+part of dart.ui;
+
+/// Functionality for Flutter plugin authors.
+abstract class PluginUtilities {
+  /// Get a closure instance for a specific static or top-level function given
+  /// a name.
+  ///
+  /// **Warning:** if the function corresponding with `name` is not referenced
+  /// explicitly (e.g., invoked, passed as a closure, etc.) it may be optimized
+  /// out of the compiled application. To prevent this, avoid hard coding names
+  /// as [String]s and  prefer to use `PluginUtilities.getNameOfFunction` to
+  /// retrieve the function's name.
+  ///
+  /// `name` is the name of the function we want a closure for. This is a
+  /// required parameter.
```

Comment:
Typical wording is 'This must not be null.'

Reply:
Done.